backend/open_api_spec.yml

```diff
@@ -0,0 +1,790 @@
+                            datasets:
+                              type: array
+                              items:
+                                $ref: "#/components/schemas/dataset_preview"
+                            revision_of:
+                              type: string
+                              description: uuid of the public version of this Collection
+                            access_type:
+                              type: string
+                              enum: [ READ, WRITE ]
+                            published_at:
+                              type: number
+                            description:
+                              type: string
+                            obfuscated_uuid:
+                              type: string
+                            data_submission_policy_version:
+                              type: string
+                            genesets:
+                              type: array
+                              items:
+                                $ref: "#/components/schemas/gene_set"
+        "401":
+          $ref: "#/components/responses/401"
+    post:
+      description: Create a new Collection
+      security:
+        - cxguserCookie: []
+      requestBody:
+        content:
+          application/json:
+            $ref: "#/components/schemas/query_collection_metadata"
+      responses:
+        "201":
+          description: Created
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/collection_uuid"
+        "400":
+          $ref: "#/components/responses/400"
+        "401":
+          $ref: "#/components/responses/401"
+
+  /curation/collections/{collection_id}:
+    delete:
+      description: Delete a private Collection. Delete an existing revision for a public Collection.
+      security:
+        - cxguserCookie: []
+      parameters:
+        - $ref: "#/components/parameters/path_collection_uuid"
+      responses:
+        "202":
+          description: Accepted
+          content:
+            application/json:
+              type: string
+        "401":
+          $ref: "#/components/responses/401"
+        "404":
+          $ref: "#/components/responses/404"
+    get:
+      description: Fetch Collection metadata and associated Datasets, public and private.
+      security:
+        - cxgUserCookieLenient: []
+      parameters:
+        - $ref: "#/components/parameters/path_collection_uuid"
+        - $ref: "#/components/parameters/query_visibility"
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                additionalProperties: false
+                required:
+                  - name
+                  - description
+                  - contact_name
+                  - contact_email
+                  - curator_name
+                properties:
```

Comment:
Should this be replaced with `allOf` and  `$ref: "#/components/schemas/query_collection_metadata"` and the extra properties `published_at`, `revised_at` and `datasets`?

Reply:
In actuality, no. Because the collection data we take in for POST/PUT of a collection is not de facto a logical subset of the stuff we would return for a GET.

reverted (`7ab414f3`)